frontend/src/CSets.js

```diff
@@ -114,19 +114,24 @@ function CSetsFromDisk(props) {
 const API_ROOT = 'http://127.0.0.1:8000'
 const enclave_url = path => `${API_ROOT}/passthru?path=${path}`
 const backend_url = path => `${API_ROOT}/${path}`
+// let url_concept_sets = enclave_url('objects/OMOPConceptSet')
+let url_concept_sets = backend_url('datasets/csets')
 
 function ConceptSets(props) {
-  let path = 'objects/OMOPConceptSet';
-  let url = enclave_url(path)
+  let url = url_concept_sets
   let navigate = useNavigate();
   const { isLoading, error, data, isFetching } = useQuery([url], () =>
       axios
           .get(url)
-          .then((res) => res.data.data.map(d => d.properties))
-  );
+          .then((res) => res.data.map(d => d.properties))
+  )
   if (isLoading) return "Loading...";
 
-  if (error) return "An error has occurred: " + error.message;
+  if (error) {
+    const err = "An error has occurred: " + error.message;
+    console.error(err)
```

Comment:
@Sigfried what you had looked really aesthetic, but did not provide the interactivity provided by `console.error` (and also probably from `console.log()`). There's also `console.debug()` and a few others; I haven't really used these before but I should try to play w/ them.